### DISCUSSION TITLE
DFU Not finding Transmitter

### DISCUSSION THREAD

**Discussion by Pinguinito0812** [FrSkyRC/ETHOS-Feedback-Community#4639] [Q&A]
Sup everyone hoping to get some help here. I have a new Frsky x18se which i was trying to update to use the stabilization mode. It will not longer turn on as well as i cannot through the DFU interface update the bootloader. Any suggestions? Thank you all

On Ethos the software is as follows
Firmware version: 1.5.15 FLEX (Up to Date)
Bootloader version: 1.4.12 (Not up to Date)
Module version: 2.2.7 (Up to Date)
Flash (8.0MB): System Bitmaps 1.5.14 (Up to Date)

**Comment by bsongis-frsky** [comment]
It doesn't turn on in firmware mode?
It doesn't turn on in bootloader mode?
Is there anything detected from the computer in DFU mode (transmitter OFF when plugged USB)?

**Reply by Pinguinito0812** [reply]
I can access BOOTLOADER mode. I can see telemetry through the ETHOS software on my computer. I can update everything BUT the bootloader. It just says DFU device not found. But it ssays the bootloader software i am prepping to use is compatible and current.
